Makefile

```diff
@@ -2,7 +2,7 @@
 # Version details                                                              #
 ################################################################################
 
-GIT_VERSION = $(shell git describe --always --abbrev=7 --dirty)
+GIT_VERSION = $(shell git describe --always --abbrev=7 --dirty --match=NeVeRmAtCh)
```

Comment:
This requires some explanation. `git describe --always --abbrev=7 --dirty` will resolve to an annotated tag if there is one and you'll get that instead of the sha. Since we always explicitly want the sha here, the little `--match=NeVeRmAtCh` hack helps ensure that the sha is what we will always get.